archive/action.yml

```
@@ -3,7 +3,12 @@ description: 'Stores artifacts'
 inputs:
   path:
     description: 'repository-relative to the files to archive'
```

```suggestion
    description: 'repository-relative path to the files to archive'
```

(i think? not part of this PR really, tho)